src/binaryen-c.cpp

```diff
@@ -190,41 +190,42 @@ void BinaryenModuleDispose(BinaryenModuleRef module) {
 
 // Function types
 
-BinaryenFunctionTypeRef BinaryenAddFunctionType(BinaryenModuleRef module, const char* name, BinaryenType result, BinaryenType* paramTypes, BinaryenIndex numParams) {
+BinaryenFunctionTypeRef BinaryenAddFunctionType(BinaryenModuleRef module, const char* name,
+  BinaryenType result, BinaryenType* paramTypes, BinaryenIndex numParams) {
   auto* wasm = (Module*)module;
-  auto* ret = new FunctionType;
-  if (name) ret->name = name;
-  else ret->name = Name::fromInt(wasm->functionTypes.size());
+  std::unique_ptr<FunctionType> ret{new FunctionType};
+  if (name)
```

Comment:
our code conventions require `{` and `}` if the if or if-else bodies are not on the same line.

Reply:
Ok, I will try to change it. But I believe I used clang-format here.

Reply:
I think we investigated that and found we couldn't do it in clang-format, and only clang-tidy can fix up such things. But I may be thinking of another notation.

Reply:
I reverted the code format change.